GOVERNANCE.md

```diff
@@ -43,3 +43,11 @@ In general, we prefer that technical issues and maintainer membership are amicab
 # Adding new projects to the Backstage GitHub organization
 
 New projects will be added to the Backstage organization via GitHub issue discussion in one of the existing projects in the organization. Once sufficient discussion has taken place (~3-5 business days but depending on the volume of conversation), the maintainers of the project where the issue was opened (since different projects in the organization may have different maintainers) will decide whether the new project should be added. See the section above on voting if the maintainers cannot easily decide.
+
+# Reviewers
+
+The project also contains a team called [@backstage/reviewers](https://github.com/orgs/backstage/teams/reviewers). This is the group of people who are the fallback in [`CODEOWNERS`](./.github/CODEOWNERS). This team will typically contain the maintainers, and a small number of additional people who are permitted to approve and merge pull requests. The purpose of this group is to offload some of the review work from the maintainers, simplifying and speeding up the review process for contributors.
+
+This responsibility is distinct from the maintainer role. A reviewer must not approve and merge changes that have significant impact on architecture, design, or important programming contact surfaces. For that class of changes, a reviewer can still review the pull request without approving it, and is expected to notify `@backstage/maintainers` for final approval.
+
+Reviewers are added and removed through the conference of the maintainers and sponsors. This is meant to be a lightweight process, and implies a certain amount of trust.
```

Comment:
I'm not sure what exactly this means. Does "conference of the maintainers and sponsors" mean that we need a majority of those groups to vote on a reviewer? Does it just mean that having 2 agree maintainers can add or remove any reviewers?

Reply:
I'd actually appreciate suggestions here, what you see is the result of me struggling a bit to formulate it :) I would like for the end result be that it's really easy for the maintainers to for example add and remove Spotify employees that pitch in but haven't gotten maintainer status.

Reply:
I think that we should have a specific definition for what the process is, but try to keep it light. At the moment, I'd suggest "a simple majority vote by the maintainers".
This vote would just be on the PR. So, someone creates a PR adding a person to the reviewers list and once it has a 👍  from a simple majority of maintainers, it's merged.

Reply:
Feedback from legal: The main thing to be clear on is the process for adding or removing reviewers (which looks like it was called out already), as well as what constitutes a PR with "significant impact" that requires maintainer approval

Reply:
Yep - I have been slow in updating this PR but it's on my todo

Reply:
Updated! Let me know if there's something that needs adjusting here